frontend/public/components/utils/managed-by.tsx

```diff
@@ -0,0 +1,116 @@
+import * as React from 'react';
+import * as classNames from 'classnames';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+
+import { ResourceIcon } from './resource-icon';
+import {
+  K8sResourceCommon,
+  K8sResourceKind,
+  referenceForGroupVersionKind,
+  referenceForModel,
+  referenceForOwnerRef,
+  OwnerReference,
+} from '../../module/k8s';
+import { RootState } from '@console/internal/redux';
+import { withFallback } from '@console/shared/src/components/error/error-boundary';
+import { ClusterServiceVersionModel } from '@console/operator-lifecycle-manager/src/models';
+import { ClusterServiceVersionKind } from '@console/operator-lifecycle-manager';
+import { k8sList } from '../../module/k8s/resource';
+
+const managedByLinkStateToProps = (state: RootState): managedByLinkStateProps => {
+  return {
+    activeNamespace: state.UI.get('activeNamespace'),
```

Comment:
I don't think you need to connect the component. Don't you have the `obj` which has the namespace set in it?